### PR TITLE
[DNS] Preparing to release the latest version

### PR DESCRIPTION
**Pull request by melina5656**

### CODE REVIEW

File: sdk/dns/Azure.ResourceManager.Dns/api/Azure.ResourceManager.Dns.netstandard2.0.cs [hunk 1]
```diff
@@ -316,18 +399,52 @@ public partial class DnsRecordData : Azure.ResourceManager.Dns.DnsBaseRecordData
...
+        void System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Dns.DnssecConfigData>.Write(System.Text.Json.Utf8JsonWriter writer, System.ClientModel.Primitives.ModelReaderWriterOptions options) { }
+        Azure.ResourceManager.Dns.DnssecConfigData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Dns.DnssecConfigData>.Create(System.BinaryData data, System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Dns.DnssecConfigData>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Dns.DnssecConfigData>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+    }
+    public partial class DnssecConfigResource : Azure.ResourceManager.ArmResource, System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Dns.DnssecConfigData>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Dns.DnssecConfigData>
```
Comment (ArthurMa1978) [inline sdk/dns/Azure.ResourceManager.Dns/api/Azure.ResourceManager.Dns.netstandard2.0.cs:429]: Is this `DnsSec` ?

File: sdk/dns/Azure.ResourceManager.Dns/api/Azure.ResourceManager.Dns.netstandard2.0.cs [hunk 2]
```diff
@@ -404,6 +521,45 @@ public partial class DnsSrvRecordResource : Azure.ResourceManager.ArmResource
         public virtual Azure.Response<Azure.ResourceManager.Dns.DnsSrvRecordResource> Update(Azure.ResourceManager.Dns.DnsSrvRecordData data, Azure.ETag? ifMatch = default(Azure.ETag?), System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.Dns.DnsSrvRecordResource>> UpdateAsync(Azure.ResourceManager.Dns.DnsSrvRecordData data, Azure.ETag? ifMatch = default(Azure.ETag?), System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
     }
+    public partial class DnsTlsaRecordCollection : Azure.ResourceManager.ArmCollection, System.Collections.Generic.IAsyncEnumerable<Azure.ResourceManager.Dns.DnsTlsaRecordResource>, System.Collections.Generic.IEnumerable<Azure.ResourceManager.Dns.DnsTlsaRecordResource>, System.Collections.IEnumerable
```
Comment (ArthurMa1978) [inline sdk/dns/Azure.ResourceManager.Dns/api/Azure.ResourceManager.Dns.netstandard2.0.cs:524]: What does `Tlsa` short for? Or this is `DnsTlsARecord`?

File: sdk/dns/Azure.ResourceManager.Dns/samples/autorest.tests.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Generated code configuration
```
Comment (ArthurMa1978) [inline sdk/dns/Azure.ResourceManager.Dns/samples/autorest.tests.md:1]: this file was retired

**Review by ArthurMa1978 — CHANGES_REQUESTED** [review]
Update the changelog.md and Azure.ResourceManager.Dns.csproj files for releasing

File: sdk/dns/Azure.ResourceManager.Dns/api/Azure.ResourceManager.Dns.netstandard2.0.cs (hunk 2 above)
Comment (melina5656) [reply sdk/dns/Azure.ResourceManager.Dns/api/Azure.ResourceManager.Dns.netstandard2.0.cs:524]: Tlsa represents Transport Layer Security Authentication

File: sdk/dns/Azure.ResourceManager.Dns/api/Azure.ResourceManager.Dns.netstandard2.0.cs (hunk 1 above)
Comment (melina5656) [reply sdk/dns/Azure.ResourceManager.Dns/api/Azure.ResourceManager.Dns.netstandard2.0.cs:429]: Its full name is DNS Security extensions (DNSSEC), so Dnssec should be reasonable.

**Review by ArthurMa1978 — APPROVED** [review]

**Pull request merged**